src/mono/wasm/runtime/method-calls.ts

```diff
@@ -626,6 +626,7 @@ export function mono_wasm_compile_function_ref(code: MonoStringRef, is_exception
 
     const js_code = conv_string_root(codeRoot);
     if (!js_code) {
+        // FIXME: We should probably throw an error here, compiling an empty function is almost certainly a mistake
```

Comment:
Let's delete the whole thing including `CompileFunctionRef` in C#. It's unused AFAIK.